modules/ObjectControl/inc/objectcontrol.hpp

```diff
@@ -70,6 +77,7 @@ namespace AbsoluteKinematics {
 	class Disarming;
 	class Done;
 	class RemoteControlled;
+	class Resetting;
```

Comment:
Why is a new Resetting state needed when there is a isResetting member variable?

Reply:
Remains from another approach that I forgot to remove. Fixed now